spec/features/contestant/index_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "user story 2" do
+  before(:each) do
+    test_data
+  end
+  describe "the contestant index page" do
+    it "displays a list of names of all contestants" do
+      visit "/contestants"
+
+      expect(page).to have_content(@jay.name)
+      expect(page).to have_content(@gretchen.name)
+      expect(page).to have_content(@kentaro.name)
+      expect(page).to have_content(@erin.name)
+    end
+
+    # it "displays all projects a contestant has done beneath their name" do
+    #I don't know how to do this'
+    #   visit "/contestants"
+
+    #   expect(page).to have_content(Projects: )
```

Comment:
I see you were trying stuff out in the views, but try to stick with TDD and write your tests _regardless_ of whether or not you know how to implement it. There's lots of reasons for this, but the main one for you as you keep learning Rails is, if your tests are right you'll be able to figure out the code as you go. If you try to figure out the code before writing the test, its' really easy to get distracted or go down the wrong rabbit hole (and there are many).

Reply:
I recommend coming back to this test and finishing it out for good practice; same with user story 3!